src/integrationTest/java/uk/gov/hmcts/darts/event/testutils/DartsDatabaseStub.java

```diff
@@ -35,8 +37,11 @@ public class DartsDatabaseStub {
     MediaRepository mediaRepository;
     ExternalObjectDirectoryRepository externalObjectDirectoryRepository;
 
+    NotificationRepository notificationRepository;
```

Comment:
Can this field (and the others) be made `private` and  `final`?

Reply:
done